apps/graphql/test/web/resolvers/capitation_contract_request_resolver_test.exs

```
@@ -788,23 +813,21 @@ defmodule GraphQLWeb.CapidationContractRequestResolverTest do
         "text" => "something"
       }
 
-      assert capture_log(fn ->
-               resp_body =
-                 conn
-                 |> put_client_id(legal_entity.id)
-                 |> put_consumer_id(party_user.user_id)
-                 |> put_req_header("drfo", legal_entity.edrpou)
-                 |> put_scope("contract_request:update")
-                 |> post_query(@approve_query, input_signed_content(contract_request.id, content))
-                 |> json_response(200)
-
-               refute resp_body["data"]["approveContractRequest"]
-
-               assert match?(
-                        %{"message" => "Undefined error", "extensions" => %{"code" => "BAD_REQUEST"}},
-                        hd(resp_body["errors"])
-                      )
-             end) =~ "Got undefined error"
+      resp_body =
+        conn
+        |> put_client_id(legal_entity.id)
+        |> put_consumer_id(party_user.user_id)
+        |> put_req_header("drfo", legal_entity.edrpou)
+        |> put_scope("contract_request:update")
+        |> post_query(@approve_query, input_signed_content(contract_request.id, content))
+        |> json_response(200)
+
+      refute resp_body["data"]["approveContractRequest"]
```

Suppose that there should be 2 checks
`assert Map.has_key?(resp_body["data"], "approveContractRequest")`
`refute resp_body["data"]["approveContractRequest"]`

---

There always be such key as long as such mutation is defined on the schema

---

For example that would passed
`refute resp_body["data"]["approveContractRequestt"]`
or that `refute resp_body["data"]["approveContractReq..."]`

---

In case if such mutation is absent on the schema, our request will fail early with a request validation error

---

ok